README.md

```diff
@@ -392,6 +392,110 @@ The default port the Pushgateway is listening to is 9091. The path looks like:
 * For example to wipe all metrics from the Pushgateway:
 
         curl -X PUT http://pushgateway.example.org:9091/api/v1/admin/wipe
+
+## Rest API
```

Comment:
This API is not really RESTful, or at least the push API above is more REST-like than this one.

How about _Query API_?